src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -210,6 +218,24 @@ export default class PaypalCommercePaymentProcessor {
         const isAPM = this._gatewayId === PaymentStrategyType.PAYPAL_COMMERCE_ALTERNATIVE_METHODS;
         const { orderId } = await this._paypalCommerceRequestSender.setupPayment(cartId, {...paramsForProvider, isAPM});
         this._orderId = orderId;
+        if (this._fundingSource === 'oxxo') {
+            const paymentData =  {
+                formattedPayload: {
+                    vault_payment_instrument: null,
+                    set_as_default_stored_instrument: null,
+                    device_info: null,
+                    method_id: this._fundingSource,
+                    paypal_account: {
+                        order_id: this._orderId,
+                    },
+                },
+            };
+            const payload = this.getOrderRequestBody(this._gatewayId, this._fundingSource, paymentData);
+            const { payment, ...order } = payload;
+            const paymentRequestOptions = this.getPaymentRequestOptions(this._gatewayId, this._fundingSource);
+            await this._store.dispatch(this._orderActionCreator.submitOrder(order, paymentRequestOptions));
```

Comment:
I think you should check the `options` argument of `_orderActionCreator.submitOrder` method. As far as I know, it should contain only `timeout` and `params` data.

Reply:
Perhaps, you can pass it like:
```suggestion
            await this._store.dispatch(this._orderActionCreator.submitOrder(
                order, 
                { params: paymentRequestOptions }
            ));
```